config/import-control.xml

```diff
@@ -16,6 +16,18 @@
   <allow pkg="org.apache.commons.logging"/>
   <allow pkg="org.xml.sax"/>
 
+  <!--Disallowed until https://github.com/mojohaus/cobertura-maven-plugin/issues/29-->
+  <disallow class="java.util.stream.Stream"/>
```

Comment:
@MEZk , as you claim it is related to all other Stream classes in this package "java.util.stream", lets disallow all of them (not that much of them in package)